### PR TITLE
EZP-29986: Content model revision

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-29986
| Versions      | all

### CODE REVIEW

File: docs/guide/content_management.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+!!! note
+
+    Displaying visible or hidden Locations in governed by the [`Visibility` Search Criterion](../api/public_php_api_browsing.md#criterionvisibility)
+
+!!! caution "Visibility and permissions"
+
+    The Location visibility flag is not permission-based and thus acts as a simple potential filter.
+    **It is not meant to restrict access to content**.
+
+    If you need to restrict access to a given Content item, use **Sections** or other **Limitations**, which are permission-based.
```
Comment (MagdalenaZuba) [inline docs/guide/content_management.md]: Please add links to Limitations and Sections

File: docs/guide/content_management.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+Content items are located in a tree structure through the Locations they are placed in.
+However, Content items themselves can also be related to one another.
+
+A **Relation** can exist between any two Content items in the Repository.
+For example, images are linked to news articles they are used in.
+Instead of using a fixed set of image attributes, the images are stored as separate Content items outside the article.
+
+There are different types of Relations available in the system.
+Content can be related on item or on Field level.
```
Comment (MagdalenaZuba) [inline docs/guide/content_management.md]: can be related to?

File: docs/guide/content_management.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+*Relations at Content item level* can be of three different types:
+
+- *Common relations* are created between two Content items using the Public API.
+- *RichText linked relations* are created using a Field of the RichText type.
+When an internal link (a link to another Location or Content item) is placed in a RichText Field,
+the system automatically creates a Relation.
+The Relation is automatically removed from the system when the link is removed from the Content item.
+- *RichText embedded relations* also use a RichText Field.
+When an Embed element is placed in a RichText Field, the system automatically creates a Relation
+between the embedded Content item to the one with the RichText Field.
```
Comment (MagdalenaZuba) [inline docs/guide/content_management.md]: **between** the embedded Content item **and** the one with the RichText Field.

File: docs/guide/content_model.md [hunk 4]
```diff
@@ -2,41 +2,71 @@
...
 
-    Content item Name is always searchable, even if the Field(s) used to generate it are not.
+    `name` is always searchable, even if the Field(s) used to generate it are not.
+
+**`sectionId`** - the unique number of the Section to which the Content item belongs.
+New Content items are placed in the Standard section by default.
+This behavior can be changed, but content must always belong to some Section.
+See [Sections](admin_panel.md#sections) for more information.
+
+**`currentVersionNo`** - current version number is the version number of the published version or the version number of a newly created draft (which is 1).
```
Comment (MagdalenaZuba) [inline docs/guide/content_model.md]: of the published content? (too many versions :D)

File: docs/guide/content_model.md [hunk 5]
```diff
@@ -2,41 +2,71 @@
...
-**Section ID** – the unique number of the Section to which the Content item belongs. New Content items are placed in the Standard section by default. This behavior can be changed, but content must always belong to some Section.
+**`remoteId`** - global unique ID of the Content item.
 
-**Versions** – total number of versions of this Content item. A new version is created every time the Content item is modified and it is this new version that is edited. The previous / published version along with earlier versions remain untouched and its status changed to *Archived*. A Content item always has at least one version. 
+**`mainLanguageCode`** - the main language code of the Content item.
+If the `alwaysAvailable` flag is set to true, the Content item is shown in this language when the requested language does not exist.
+
+**`mainLocationId`** - identifier of the Content item's main [Location](content_management.md#locations).
+
+**`status`** - status of the Content item. It can have three statuses: 0 – Draft, 1 – Published and 2 – Archived.
```
Comment (MagdalenaZuba) [inline docs/guide/content_model.md]: `0 – *draft*, 1 – *published* and 2 – *archived*.`

File: docs/guide/content_model.md [hunk 6]
```diff
@@ -80,31 +110,40 @@ Each Content Type is characterized by a set of metadata which define the general
...
 
 ![Creating a new Content Type](img/admin_panel_new_content_type.png)
 
 ### Field definitions
 
-Aside from the metadata, a Content Type contains any number of Field definitions (but has to contain at least one). They determine what Fields of what Field Types will be included in all Content items based on this Content Type.
-
-A Content Type and its Field definitions can be modified after creation, even if there are already Content items based on it in the system. When a Content Type is modified, each of its instances will be changed as well. If a new Field definition is added to a Content Type, this Field will appear (empty) in every relevant Content item. If a Field definition is deleted from the Content Type, all the corresponding Fields will be removed from Content items of this type.
+Aside from the metadata, a Content Type contains any number of Field definitions (but has to contain at least one).
```
Comment (MagdalenaZuba) [inline docs/guide/content_model.md]: a Content Type may contain any number

File: docs/guide/content_model.md [hunk 7]
```diff
@@ -124,8 +163,23 @@ Note that the Required flag is in no way related to Field validation. A Field's
...
+**[Translatable](internationalization.md)** – a flag which indicates if the value of the Field can be translated. This is independent of the Field Type, which means that even Fields such as "Float" or "Image" can be set as translatable.
 
 Depending on the Field Type, there may also be other, specific information to fill in. For example, the "Country" Field Type allows you to select the default country, as well as to allow selecting multiple countries at the same time.
 
 ![Diagram of content model](img/content_model_diagram.png)
+
+## Content versions
+
+Each Content item can have multiple versions.
+Each version has one of the following statuses: *Draft*, *Archived* or *Published*.
```
Comment (MagdalenaZuba) [inline docs/guide/content_model.md]: *draft*, *archived* or *published*.

File: docs/guide/content_management.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+
+!!! tip "In short"
+
+    A Location can only be Visible when all of its ancestors are Visible as well.
+
+#### Visibility mechanics
+
+The visibility mechanics are controlled by two flags: Hidden flag and Invisible flag.
+The Hidden flag informs whether the node has been hidden by a user or not.
+A raised Invisible flag means that the node is invisible either because it was hidden by a user or by the system.
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/content_management.md]: Actually, the Invisible flag is also raised (1) when "The Location is invisible and it was hidden by a user."

**Pull request merged**